src/content/1.7/modules/creation/adding-configuration-page.md

```diff
@@ -320,3 +320,100 @@ in the back office:
 Change the value to whichever you like, click on the "Save" button, then
 go reload the homepage: your module is indeed updated with the new
 string!
+
+Using HelperOptions
+----------------
+
+THe `HelperOptions` helper is the recommanded one for managing configurations i.e. values stored in the configuration table.
```

Comment:
Where is it recommended ?

Reply:
it's recommended by @Matt75

Reply:
@Progi1984 you can see it there in action https://github.com/Matt75/displayordercarrier/blob/master/controllers/admin/AdminDisplayOrderCarrierController.php

Reply:
I wouldn't say that it is a recommended way as it's legacy part of the code, it is for legacy part, but for sure not for a modern architecture :) we need to inform developers about it

Reply:
Thanks @kpodemski I've changed this.